packages/main/src/Icon.ts

```diff
@@ -327,6 +328,16 @@ class Icon extends UI5Element implements IIcon {
 	get hasIconTooltip() {
 		return this.showTooltip && this.effectiveAccessibleName;
 	}
+
+	get accessibilityInfo() {
+		if (this.mode === IconMode.Interactive) {
+			return {
+				role: this.effectiveAccessibleRole as AriaRole,
```

Comment:
The `type` is missing. The user deserves to know the component type. Please refer to sap/m/Image#getAccessibilityInfo.